.github/workflows/terraform.yml

```diff
@@ -8,24 +8,40 @@ on:
 
 jobs:
   terraform:
-    name: 'Terraform'
+    name: ${{ matrix.name }}
     runs-on: ubuntu-latest
     
     defaults:
       run:
-        working-directory: terraform/global
+        working-directory: terraform/${{ matrix.env }}
     
     env:
       AWS_ACCESS_KEY_ID:     ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AWS_DEFAULT_REGION:    ${{ secrets.AWS_DEFAULT_REGION }}
-      TF_VAR_access_key:     ${{ secrets.AWS_ACCESS_KEY_ID }}
-      TF_VAR_secret_key:     ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      TF_VAR_region:         ${{ secrets.AWS_DEFAULT_REGION }}
-      # TF_VAR_cf_user:        ${{ secrets.DEV_CF_USER }}
-      # TF_VAR_cf_password:    ${{ secrets.DEV_CF_PASSWORD }}
-      # TF_VAR_uev_key:        ${{ secrets.DEV_UEV_KEY }}
-    
+      TF_VAR_access_key:     ${{ secrets[matrix.access_key] }}
+      TF_VAR_secret_key:     ${{ secrets[matrix.secret_key] }}
+      TF_VAR_region:         ${{ secrets[matrix.region] }}
+      TF_VAR_cf_user:        ${{ secrets[matrix.cf_user] }}
+      TF_VAR_cf_password:    ${{ secrets[matrix.cf_password] }}
+
+    strategy:
+      matrix:
+        env: [global, staging]
+        include:
+          - env: global
```

Comment:
should these env variables also include `cf_user` and `cf_password` ?

Reply:
not right now, the `global` env doesn't touch CF